src/containers/Secrets/Secrets.js

```diff
@@ -410,7 +410,7 @@ export /* istanbul ignore next */ class Secrets extends Component {
             >
               <div className="secretHelper">
                 {intl.formatMessage({
-                  id: 'dashboard.secretType.helper',
+                  id: 'dashboard.secrets.secretType.helper',
```

Comment:
updated in https://github.com/tektoncd/dashboard/pull/1205/files#diff-7e87000f350345d450edee6cb843d4feR414 to use the same value as the other string with the same id (was an oversight from a recent change)

if we could get #1205 in, then rebase this and remove the Secrets changes from here that'd be great